lib/src/connection.dart

```diff
@@ -495,3 +502,76 @@ class Notification {
   /// An optional data payload accompanying this notification.
   final String payload;
 }
+
+class PostgreSQLConnectionPool {
+  PostgreSQLConnectionPool(this.size, this.host, this.port, this.databaseName,
+      {this.username: null,
+        this.password: null,
+        this.timeoutInSeconds: 30,
+        this.timeZone: "UTC",
+        this.useSSL: false});
+
+  bool _isClosed = false;
+  final int size;
+  final String host;
+  final int port;
+  final String databaseName;
+  final String username;
+  final String password;
+  final int timeoutInSeconds;
+  final String timeZone;
+  final bool useSSL;
+  var _connections = new List<PostgreSQLConnection>();
+
+
+  PostgreSQLConnection get connection {
```

Comment:
If the pool is in the middle of closing (waiting for Futures in close() to complete), this will return a connection that may or may not be valid.